.gitreview

```diff
@@ -1,6 +0,0 @@
-[gerrit]
```

Comment:
All three commits proudly carry a `Change-Id` - no harm done but we don't need it any more, can lose the hook, right?